src/django_otp_webauthn/backends.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.http import HttpRequest
+
+from django_otp_webauthn.models import AbstractWebAuthnCredential
+
+UserModel = get_user_model()
+
+class WebAuthnBackend:
+    """A simple authentication backend used when django_otp_webauthn is used for passwordless authentication"""
+
+    def authenticate(self, request: HttpRequest, web_authn_credential: AbstractWebAuthnCredential | None = None, **kwargs: Any) -> AbstractBaseUser | None:
+        if web_authn_credential:
+            user = web_authn_credential.user
+            return user if self.user_can_authenticate(user) else None
+        return None
+    
+    def get_user(self, user_id) -> AbstractBaseUser | None:
+        user = UserModel._default_manager.get(pk=user_id)
+        return user if self.user_can_authenticate(user) else None
```

Comment:
Oops. This ties right into my comment I left about this and I still didn't put it together. That was based on using `.first()`. I see little point in having the extra exception handling vs just letting `user_can_authenticate()` return `False` if there is no user since handling that situation makes sense to me, anyway.

I reorganized that code a couple of times but missed this goofiness even while commenting on it myself.